checkov/common/output/record.py

```diff
@@ -208,6 +210,24 @@ def to_string(self, compact: bool = False, use_bc_ids: bool = False) -> str:
         else:
             return f"{check_message}{status_message}{severity_message}{detail}{file_details}{caller_file_details}{evaluation_message}{guideline_message}"
 
+    def _get_secret_validation_status_message(self) -> str:
+        warning_sign_unicode = '\u26a0'
+        text_by_secret_validation_status_status = {
+            ValidationStatus.VALID.value: colored(f'\t{warning_sign_unicode} This secret has been validated'
+                                                  f' and should be prioritized\n', "red"),
+            ValidationStatus.INVALID.value: '\tThis is not a valid secret and can be de-prioritized\n',
+            ValidationStatus.UNKNOWN.value: '\tWe were not able to validate this secret\n',
+            ValidationStatus.UNAVAILABLE.value: ''
+        }
```

Comment:
maybe this can be declared once as a const? :)
also maybe better to move it to the secrets record file?

Reply:
Moved it to secrets_record :)
I don't think it should be a const as there's no other function that needs it

Reply:
since you invoke the function each time a secret verifier status is needed, the dict will be recreated. therefore moving it to a const, like @marynaKK suggested, makes it more performant. Also is there any reason why `warning_sign_unicode` is a variable and not just inlined?

Reply:
Good point, will move to const.

It's a variable just for readability's sake